wolfssl/ssl.h

```diff
@@ -2413,8 +2413,12 @@ WOLFSSL_ABI WOLFSSL_API void wolfSSL_X509_free(WOLFSSL_X509*);
 /* get index cert in PEM */
 WOLFSSL_API int  wolfSSL_get_chain_cert_pem(WOLFSSL_X509_CHAIN*, int idx,
                                 unsigned char* buf, int inLen, int* outLen);
+WOLFSSL_API int wolfSSL_get_sessionIDSz(const WOLFSSL_SESSION* session);
```

Comment:
If these new API's only apply to `SESSION_CERTS` can you wrap the headers in it as well?